commit d2c9455ceead58427ca77c8be470d924de5f6913
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Sep 17 17:31:18 2018 +0100

    cxd2880: CXD2880_SPI_DRV should select DVB_CXD2880 with MEDIA_SUBDRV_AUTOSELECT

diff --git a/drivers/media/spi/Kconfig b/drivers/media/spi/Kconfig
index b07ac86fc53c..f76dea36ab8a 100644
--- a/drivers/media/spi/Kconfig
+++ b/drivers/media/spi/Kconfig
@@ -19,6 +19,7 @@ menu "Media SPI Adapters"
 config CXD2880_SPI_DRV
 	tristate "Sony CXD2880 SPI support"
 	depends on DVB_CORE && SPI
+	select DVB_CXD2880 if MEDIA_SUBDRV_AUTOSELECT
 	default m if !MEDIA_SUBDRV_AUTOSELECT
 	help
 	  Choose if you would like to have SPI interface support for Sony CXD2880.
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1241_d2c9455ceead58427ca77c8be470d924de5f6913.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1241_d2c9455ceead58427ca77c8be470d924de5f6913.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
